docs/auth/android.md

```diff
@@ -19,19 +19,20 @@ Add the `RNFirebaseAuthPackage` to your `android/app/src/main/java/com/[app name
 
 ```java
 // ...
-import io.invertase.firebase.RNFirebasePackage;
+import com.facebook.react.ReactApplication;
 import io.invertase.firebase.auth.RNFirebaseAuthPackage; // <-- Add this line
 
 public class MainApplication extends Application implements ReactApplication {
     // ...
 
     @Override
     protected List<ReactPackage> getPackages() {
-      return Arrays.<ReactPackage>asList(
-          new MainReactPackage(),
-          new RNFirebasePackage(),
-          new RNFirebaseAuthPackage() // <-- Add this line
-      );
+      @SuppressWarnings("UnnecessaryLocalVariable")
+      List<ReactPackage> packages = new PackageList(this).getPackages();
+      // Packages that cannot be autolinked yet can be added manually here, for example:
+      // packages.add(new MyReactNativePackage());
+      packages.add(new RNFirebaseAuthPackage()); <-- Add this line
```

Comment:
this isn't an actual comment - should compile if cut-n-pasted just in case
```suggestion
      packages.add(new RNFirebaseAuthPackage()); // <-- Add this line
```

Reply:
Done